pkg/arvo/sys/vane/ames.hoon

```diff
@@ -3902,12 +3901,17 @@
           ::
           ++  fi-unsub
             |=  [=^duct all=?]
-            ?.  |(all (~(has in listeners.keen) duct))
-              %.  fine
-              (fi-trace fin.veb |.("unknown listener, skip"))
-            =?  fine  all  (~(rep in listeners.keen) fi-miss)
-            %-  (fi-trace fin.veb |.("deleting {<fi-full-path>}"))
-            fine(listeners.keen ?:(all ~ (~(del in listeners.keen) duct)))
+            ^+  fine
+            ?:  all
+              %-  (fi-trace fin.veb |.("unsub all {<fi-full-path>}"))
+              (~(rep in listeners.keen) (fi-give-tune ~))
```

Comment:
we need to delete all the listeners here so +fi-abet removes the path from the keen state

Reply:
Thanks, that's my bad.